src/Nest/Aggregations/Bucket/SignificantTerms/SignificantTermsAggregation.cs

```diff
@@ -131,7 +131,7 @@ public class SignificantTermsAggregation : BucketAggregationBase, ISignificantTe
 		/// <inheritdoc />
 		public int? ShardSize { get; set; }
 
-		[Obsolete("Use MinimumDocumentCountAsLong. Fixed in NEST 6.x")]
+		[Obsolete("Use MinimumDocumentCountAsLong. Removed in NEST 6.x")]
```

Comment:
I think fixed is a better description IMO; MinimumDocumentCount is a long? in 6.x and MinimumDocumentCountAsLong is removed, since it served only as a non-breaking BWC change

Reply:
:+1: